src/components/DocFAQPage.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import Faq from "react-faq-component";
+
+const data = {
+    title: "Doctor FAQ",
+    rows: [
+        {
+            title: "Lorem ipsum dolor sit amet,",
+            content: `Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sed tempor sem. Aenean vel turpis feugiat,
+              ultricies metus at, consequat velit. Curabitur est nibh, varius in tellus nec, mattis pulvinar metus.
+              In maximus cursus lorem, nec laoreet velit eleifend vel. Ut aliquet mauris tortor, sed egestas libero interdum vitae.
+              Fusce sed commodo purus, at tempus turpis.`,
+        },
+        {
+            title: "Nunc maximus, magna at ultricies elementum",
+            content:
+                "Nunc maximus, magna at ultricies elementum, risus turpis vulputate quam, vitae convallis ex tortor sed dolor.",
+        },
+    ],
+};
+
+
+const styles = {
+    bgColor: 'white',
+    titleTextColor: "black",
+    rowTitleColor: "black",
+    rowContentColor: 'blue',
+    arrowColor: "red",
+};
```

Comment:
Just want to let you know we use a package called [styled-components](https://styled-components.com/) that does essentially the same thing in a better way (you can just write raw css instead of the camel case version and put in parameters if necessary). I'm not going to be super picky about this so I don't care if you change it or not, just letting you know.